person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- Hey :wave: Good job with the att_reader, however, the rentals should be in the attr_accessor so you have full read/write ability to access it. the id of each entity is unique, so this should only be in the attr_reader. kindly move the `rentals` to the attr_accessor.